README.md

```diff
@@ -144,12 +144,18 @@ For a large-scale example, see the [Darklang client/editor](https://github.com/d
 
 ## Starter-Kits
 
-*These starter kits are out of date - see [#46](https://github.com/darklang/rescript-tea/issues/46) for more details.*
-
 A list of starter-kits that get you up and running.
 
 > Feel free to extend this list!
 
+#### [celsobonutti/vite-template-rescript-tea](https://github.com/celsobonutti/vite-template-rescript-tea)
+
+* syntax: ReScript
```

Comment:
I think the syntax line is no longer relevant, so we can just remove it.

Reply:
Should we remove it for every kit or only the new one?
I feel leaving it for the outdated ones would be fine for historical context